### ISSUE TITLE
Oss 2week team project

### ISSUE THREAD

**Issue by codms0201** [2022OSS-17/Cafe_dessert#2]
어떤 함수 구현 맡고 싶으세용..?

**Comment by Blackcode2** [comment]
일단 CRUD, 저장, 불러오기 한 사람이 만들고 나머지는 다른 사람이 만들고 이런식이면 될거 같은데
편하신거 먼저 하시면 될거같아요

**Comment by Blackcode2** [comment]
그리고 서로 하다가 막히면 도와주는 식이면 되지 않을까 싶네요

**Comment by codms0201** [comment]
오오 좋아용!!

**Comment by codms0201** [comment]
CRUD랑 저장 불러오기 이거 저희가 했던 과제랑 거의 똑같게 하면 되겄죵??

**Comment by Blackcode2** [comment]
네 조금만 수정하면 될것 같네요

**Comment by codms0201** [comment]
보니까, 검색기능,파일저장 로딩 기능을 제외한 모든 함수를 적절히 분배하라고 하셨더라구용

**Comment by codms0201** [comment]
제가 일단 CRUD랑 저장 로딩 이거 해볼게용 이거 하구 남은 함수 똑같이 나누어서 해보는거 어떄요?? crud랑 저장, 로딩은 거의 할거 없으니까 이거 제가 한거 괜찮은지 확인 해주시면 될것 같아용

**Comment by Blackcode2** [comment]
네 알겠습니다. 저도 급한 과제 이제 끝나서 오늘 내일 시작해 볼게요

**Comment by codms0201** [comment]
넹!!그럼 제가 올려둔 코드들 보시고 수정 부분 없으면 만들 함수 분배해봐요!

**Comment by Blackcode2** [comment]
검색 기능 3개는 일단 제가 만들고 메뉴랑 장바구니 결제 유형 중에 원하시는거 하시고 남는거 제가 또 할게요

**Comment by codms0201** [comment]
넵~!

**Comment by codms0201** [comment]
CRUD 문제 딱히 없었나요오??

**Comment by Blackcode2** [comment]
제가 밖이라 아직 확인은 못했는데 들어가서 확인 해 보도록 하겠습니다.

**Comment by Blackcode2** [comment]
죄송해요. 제가 다른 과제 때문에 워낙 정신이 없어서 제대로 신경을 못 썼네요.

**Comment by codms0201** [comment]
괜찮아요~ 일요일까지 확인해주세용~!

**Comment by Blackcode2** [comment]
올려주신 코드 봤는데 주석 추가랑 수정 조금 했고 아직 수정 사항 보여서 제가 차례로 수정해서 커밋하고 푸쉬 할게요

**Comment by codms0201** [comment]
넹!! 감사합니다!

**Comment by Blackcode2** [comment]
수정할게 많아서 아직 하고 있는데 거의 다 했어요. 아마 밥먹고 와서 다 끝날거 같아요

**Comment by Blackcode2** [comment]
수정할게 많아서 오래 걸렸네요. 저희 프로젝트에 맞게 다 수정했고 Makefile이랑 gitignore 파일도 만들어 놨습니다. 제가 이제 검색 함수 3개 만들테니 나머지 만들어 주시면 될것같아요.

**Comment by Blackcode2** [comment]
검색 함수 3개 추가 해서 올려놓았습니다.

**Comment by codms0201** [comment]
넵 감사합니다

**Comment by codms0201** [comment]
도경학우님 그러면 장바구니, 결제, 메뉴 이렇게 남은건가용?

**Comment by codms0201** [comment]
그러면 그건 가게 주인한테 보이는게 아니라 손님한테 보여지는 거죠..?

**Comment by Blackcode2** [comment]
네 언급하신 3개 메뉴는 손님용 기능이고 제 생각에는 메뉴 기능에서 보여주는건 read 함수 그대로 쓰면 될것 같고 사용자가 계속 번호를 통해서 선택하고 장바구니에 담기게 하면 될것 같아요. 이것도 select 함수 수정하면 될 것 같고 나가고 싶으면 0을 입력하는 방식이면 될것 같아요.

**Comment by Blackcode2** [comment]
2 주차 개발 일지 wiki에 추가 했어요. 위에 3개 함수가 만들기 조금 까다로울 수 도 있을거 같아서 저도 같이 생각 좀 해볼게요.

**Comment by Blackcode2** [comment]
아마 제 생각에는 구조체를 하나 더 만들어야 할 것 같아요 손님용 메뉴를 담아야 하니까

**Comment by codms0201** [comment]
어우.... 그렇군여
그러면 일단 그 위키 파일 제가 나머지 더 추가할 거 해볼게용... 그리고 그 메뉴, 장바구니, 결제 이거를 조금 더 생각 해볼게용

**Comment by codms0201** [comment]
근데 도경학우님께서 만든신 메뉴 검색방법이, 고객분들이 사용하는거 아닐까요?

**Comment by codms0201** [comment]
애초에 저희 메뉴가 사장 메뉴랑 고객 메뉴가 구분되어 있지 않은거죠? 그럼 어떻게 해야 될까용..

**Comment by codms0201** [comment]
그냥 메뉴 조회 부분을 고객님들이 보는 그 메뉴판으로 하구

**Comment by Blackcode2** [comment]
그 부분은 저희가 임의로 정하면 될것 같아요.
지금까지 만든 부분을 그냥 손님이 쓴다고 설정 하거나 아니면 앞서 말한 것 처럼 사장이랑 손님용 메뉴를 따로 둬서 만들거나요

**Comment by Blackcode2** [comment]
메뉴 조회 함수가 이제 고객들이 보는 메뉴판으로 하면 될것 같아요. 말씀하신것첢]

**Comment by codms0201** [comment]
그러면 지금 추가해야되는 함수가

**Comment by codms0201** [comment]
장바구니랑 결제 방법인거죠?

**Comment by Blackcode2** [comment]
네 맞아요.

**Comment by codms0201** [comment]
장바구니 함수 만들고 있는데, 저희 약간 기본적으로 메뉴 몇개 구성 해놓는게 좋을것 같은데 product.txt에 혹시 만들어주실수 있나요

**Comment by Blackcode2** [comment]
메뉴에 장바구니 함수 같은거 추가 해달라는 말씀이시죠?

**Comment by codms0201** [comment]
넹넹

**Comment by codms0201** [comment]
그 도경학우님 혹시 가능하시다면
장바구니 함수 구현 해주실수 있나요...? 제가 다 못할것 같아서.. 일단 제가 결제 함수는 만들어 놓긴했어요.

**Comment by codms0201** [comment]
sumprice는 장바구니에서 담은 물건들의 가격합 입니다

**Comment by Blackcode2** [comment]
네 알겠습니다.

**Comment by codms0201** [comment]
감사합니다
제가 wiki 작성하고 있겠습니다

**Comment by Blackcode2** [comment]
장바구니 선택이랑 담는거 함수 추가 했어요. 이것만 해도 함수 10개 넘어가서 결제 방법은 뺐어요 추가하면 다 완성하는데 시간이 걸려서

**Comment by Blackcode2** [comment]
일단 위키 먼저 작성해야 할거 같아요
시간이 없어서

**Comment by codms0201** [comment]
아이고 죄송합니다 제가 작성하려던 위키 부분이 함수설명인데 학우님이 완성하고 그거 캡쳐해서 쓰는 부분이라 쓰지 못하고 있었는데 학우님께서 대신 해주셨네요... 다음주 부분은 제가 위키 다 작성하겠습니다..
